qtp_target_gene/validate.py

```diff
@@ -22,8 +24,48 @@
     'FASTA_Sanger': ({'raw_fasta'}, set()),
 }
 
+MUST_GZ = [
+    # raw input files: FASTQ, per_sample_FASTQ
+    'raw_forward_seqs', 'raw_barcodes', 'raw_reverse_seqs', 'raw_fasta',
+    # preprocessed files: demultiplexed, trimmed
+    'preprocessed_fastq', 'preprocessed_fasta']
```

Comment:
Minor but this would be better as a set:

```suggestion
MUST_GZ = {
    # raw input files: FASTQ, per_sample_FASTQ
    'raw_forward_seqs', 'raw_barcodes', 'raw_reverse_seqs', 'raw_fasta',
    # preprocessed files: demultiplexed, trimmed
    'preprocessed_fastq', 'preprocessed_fasta'}
```